mosaic-tty/src/commonMain/c/mosaic-test-tty-windows.c

```diff
@@ -65,11 +66,46 @@ MOSAIC_EXPORT MosaicTestTtyInitResult mosaic_test_init(bool stdinIsTty, bool std
 		goto err_conout;
 	}
 
-	HANDLE conoutPipeRead;
-	HANDLE conoutPipeWrite;
-	if (unlikely(!CreatePipe(&conoutPipeRead, &conoutPipeWrite, NULL, 0))) {
+	HANDLE conoutPipeEvent = CreateEvent(NULL, TRUE, FALSE, NULL);
+	if (unlikely(conoutPipeEvent == INVALID_HANDLE_VALUE)) {
 		result.error = GetLastError();
-		goto err_conout_pipe;
+		goto err_conout;
+	}
+
+	LPCSTR pipename = "\\\\.\\pipe\\mynamedpipe";
```

Comment:
Is it okay if pipe has such a name?

Reply:
It's just for testing. This is copy/paste from the sample.

The real one will use computed names with a counter.